openapi.proxy/docker/compose/docker-compose.yaml

```diff
@@ -10,6 +10,7 @@ services:
     environment:
       ZILLA_INCUBATOR_ENABLED: true
     volumes:
+      - ../../specs/schema.yaml:/etc/zilla/schema.yaml
```

Comment:
Let's rename `schema.yaml` to `petstore.yaml` so it is more readable and less abstract.